src/assets/styles/app.css

```diff
@@ -0,0 +1,380 @@
+*{
```

Comment:
You should ideally use some variables to store colours and other variable style values used across the file